build.gradle

```diff
@@ -170,14 +170,6 @@ subprojects {
                     url 'https://github.com/Islandora-CLAW/CLAW/issues'
                 }
 
-                distributionManagement {
```

Comment:
I'd actually keep this block around, but you might need to change some of the values.